content/recipes/reusability/miappe.md

```diff
@@ -0,0 +1,429 @@
+# Plant genomic and genetic variation data submission to EMBL-EBI databases
+
+[TOC]
+
+
+## Main Objective
+
+Plant genotyping and phenotyping data are often used in combination to make evidence-based inferences about different trait expressions. A challenge here is not necessarily to collect the data, but to offer them in a stable way over the long term in public repositories with sufficient metadata information to make conclusions about them in line with FAIR principles. Since these repositories are often not directly linked, it is even more important to provide metadata that allows users to recognise these links by their identifiers. A key point here is sample management; the identifiers assigned here help both humans and machines to understand which experimental data are linked.
+
+The main objective of the recipe is to provide a means of submitting and tracking genotyping data in public repositories, with a particular focus on plants. This includes:
+
+    1) Submission of sample data and metadata information to BioSamples.
+    2) Submission of sequencing data and metadata to ENA.
+    3) Retrieval of the correct genome assembly for the genotyping experiment
+    4) Conversion of the resulting analysis file (in VCF format) to be FAIR
+    5) Submission of the genotyping results to EVA.
+
+
+
+## Summary
+
+This recipe provides guidance for submitting plant genotyping data to public repositories. It explains step-by-step which work should be done and when. Special attention should be paid to the metadata maintenance of the data that will be deposited in different repositories as part of this recipe. A prerequisite for fully understanding this recipe is a basic knowledge of the [MIAPPE standard](https://fairsharing.org/bsg-s000543/). 
+
+The exact listing of the metadata fields required for a FAIRification of the genotyping data set within a VCF file is also part of this recipe with examples and explanations (See details in Section 4.2).
+
+## Graphical overview of the Recipe FAIRification Objectives
+![](https://i.imgur.com/iqdjWqo.png)
+
+
+
+## Capability & Maturity Table
+|Capability|Initial Maturity Level|Final Maturity Level|
+|--- |--- |--- |
+|Interoperability|minimal|repeatable|
```

Comment:
```suggestion
## Capability & Maturity Table
|Capability|Initial Maturity Level|Final Maturity Level|
|--- |--- |--- |
|Interoperability|minimal|repeatable|
```

TODO: check with @iemam and perform a mapping against the dataset maturity indicators https://github.com/FAIRplus/Data-Maturity/blob/master/docs/Indicators.md

to clearly identify which are the maximum level of maturity reached if one applies to recipe.
[F+MM-2.3R] : machine readable metadata
[F+MM-3.3C]: dataset is given of global identifier 
could be reached but I don't think there is evidence in the recipe of the following:
[F+MM-4.1R]: Metadata is represented in a semantic machine interpretable form. (is there a jsonld for the same information?)

Reply:
The levels [F+MM-2.3R] and [F+MM-3.3C] are reached, but [F+MM-4.1R] is not. 
There is a json schema: https://www.ebi.ac.uk/biosamples/schemas/certification/plant-miappe.json  , but no jsonld.